### DISCUSSION TITLE
nonebot2 `fastapi_reload_delay`设置无效（？

### DISCUSSION THREAD

**Discussion by shoucandanghehe** [nonebot/discussions#290] [问答] [answered]
我在`.env`文件里设置了
`
FASTAPI_RELOAD_DELAY=10000.0
`
但是重载时间并没有什么变化（？

**Comment by yanyongyu** [comment, marked as answer]
如果日志里显示的config里有fastapi_reload_delay的话，那就是直接传给了uvicorn，nb并没有进行其他操作

**Reply by shoucandanghehe** [reply]
01-04 12:45:22 [DEBUG] nonebot | Loaded Config: {'driver': '~fastapi', 'host': IPv4Address('0.0.0.0'), 'port': 8080, 'log_level': 'DEBUG', 'api_timeout': 30.0, 'superusers': {'******'}, 'nickname': set(), 'command_start': {'', '/'}, 'command_sep': {'.'}, 'session_expire_timeout': datetime.timedelta(seconds=120), 'environment': 'dev', 'fastapi_reload_delay': 10000.0, 'fastapi_reload': True}
🤔

**Reply by yanyongyu** [reply]
看了下uvicorn源码，watchfiles适配没有写delay，只有watchgod和statreload有用

**Reply by yanyongyu** [reply]
你可以去uvicorn仓库提issue，也许是忘记写了

**Reply by shoucandanghehe** [reply]
encode/uvicorn#1832
